docs/source/user_guide/server_types.rst

```diff
@@ -0,0 +1,38 @@
+.. _user_guide_server_types:
+
+===========================
+Client-server communication
+===========================
+
+DPF is based on a client-server architecture.
+
+The DPF Server acts as a server, while the different client APIs (CPython, IronPython, C++...)
+act as clients connecting to it.
+PyDPF is for example a term encompassing both the CPython client
+(the Python packages such as ansys-dpf-core or ansys-dpf-post, available on PyPI)
+and the IronPython client (available within Ansys Mechanical).
+
+The communication logic with a DPF server is defined when starting it using
+an instance of the :class:`ansys.dpf.core.server_factory.ServerConfig` class.
+Different predefined server configurations are available in DPF,
+to better answer different use-cases
```

Comment:
I don't understand "better" here

Reply:
@cbellot000 "better" might the wrong word. What I meant is "adapted"

Reply:
```suggestion
each answering a different use-case
```